### PR TITLE
allow changing statsd host, support python 3.6 and django 1.11

### PR DESCRIPTION
**Pull request by AlJohri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 [![Build Status](https://travis-ci.org/tom-dalton-fanduel/fdjangodog.svg?branch=master)](https://travis-ci.org/tom-dalton-fanduel/fdjangodog)
 
-A simple Django middleware for submitting timings and exceptions to Datadog.
+A simple Django middleware for submitting timings and exceptions to Datadog using statsd.
```
Comment (tom-dalton-fanduel) [inline README.md]: Maybe we should renamed "statsd" here to "DogStatsD" to be 100% clear https://docs.datadoghq.com/guides/dogstatsd/

File: fdjangodog/middleware.py [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+except ImportError:
+    from django.core.urlresolvers import resolve, Resolver404
 
 try:
     from django.utils.deprecation import MiddlewareMixin
     base_class = MiddlewareMixin
 except ImportError:
     base_class = object
 
+if hasattr(settings, 'FDJANGODOG_STATSD_HOST'):
```
Comment (tom-dalton-fanduel) [inline fdjangodog/middleware.py]: Could collapse this to a 1-liner with [getattr](https://docs.python.org/3.6/library/functions.html#getattr)

File: fdjangodog/middleware.py [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+    statsd_host = settings.FDJANGODOG_STATSD_HOST
+else:
+    statsd_host = "localhost"
+
+statsd = DogStatsd(host=statsd_host)
```
Comment (tom-dalton-fanduel) [inline fdjangodog/middleware.py]: A *tiny* personal nitpick, I'd prefer to keep 2 lines before class definitions ref. [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines)

**Review by tom-dalton-fanduel — CHANGES_REQUESTED** [review]
Couple of minor things but this looks pretty good to me - will be happy to merge when they're addressed!

File: fdjangodog/middleware.py (hunk 3 above)
Comment (tom-dalton-fanduel) [reply fdjangodog/middleware.py]: (NB This comment applies to the class below this line in case it's not clear)

**Comment by tom-dalton-fanduel** [comment]
One final thing, can you add a sentence in the Installation section of the README to document the new `FDJANGODOG_STATSD_HOST` setting?

**Comment by AlJohri** [comment]
didn't realize how nice Github's web editing interface has gotten. made all the requested changes through the UI!

**Review by tom-dalton-fanduel — APPROVED** [review]

**Comment by tom-dalton-fanduel** [comment]
Nice, thanks for this PR. I'll get it merged and deployed as a new version in the (very) near future

**Comment by tom-dalton-fanduel** [comment]
This is now merged and I've released fdjangodog 0.4.0.

Thanks a lot of the contribution!

**Pull request merged**